lib/platform/gitlab/index.ts

```diff
@@ -259,8 +264,8 @@ export async function initRepo({
   return repoConfig;
 }
 
-export function getRepoForceRebase(): boolean {
-  return false;
+export function getRepoForceRebase(): Promise<boolean> {
+  return Promise.resolve(config?.mergeMethod !== 'merge');
```

Comment:
rebase merge doesn't work?

Reply:
Nope, you have to rebase before merge too.